src/com/jambit/STDatabase.java

```diff
@@ -0,0 +1,22 @@
+package com.jambit;
+
+import java.io.IOException;
+import java.sql.SQLException;
+
+public class STDatabase {
+  private static Database database = null;
+
+  private STDatabase() {}
+
+  public static Database getInstance()
+      throws Database.MultipleConnectionException, IOException, SQLException,
+          Database.TimeOutException {
+    if (database == null) {
+      database = new Database();
+      database.connect();
```

Comment:
Add your following code here in this line.

  _catalogProps.load(new FileInputStream(PROPERTY_PATH));
    activeDatabaseConnection =
        DriverManager.getConnection(
            "jdbc:mysql://"
                + catalogProps.getProperty("database.hostIP")
                + ":"
                + catalogProps.getProperty("database.port")
                + "/"
                + catalogProps.getProperty("database.databaseName"),
            catalogProps.getProperty("database.username"),
            catalogProps.getProperty("database.password"));_